tldw_Server_API/tests/Sharing/test_share_token_service.py

```
@@ -0,0 +1,157 @@
+    result = await token_service.generate_token(
+        resource_type="workspace",
+        resource_id="ws-1",
+        owner_user_id=1,
+        expires_at="2020-01-01T00:00:00+00:00",
+    )
+    validated = await token_service.validate_token(result["raw_token"])
+    assert validated is None
+
+
+@pytest.mark.asyncio
+async def test_validate_token_max_uses_exceeded(token_service, repo):
```

Ruff (0.15.6)

[warning] Unused function argument: `repo`

(ARG001)